src/features/defi/providers/osmosis/components/OsmosisManager/Lp/Withdraw/components/Confirm.tsx

```diff
@@ -61,6 +63,18 @@ export const Confirm: React.FC<ConfirmProps> = ({ accountId, onNext }) => {
   const chainAdapter = getChainAdapterManager().get(chainId) as unknown as osmosis.ChainAdapter
 
   const assets = useAppSelector(selectAssets)
+  const marketData = useAppSelector(selectMarketDataSortedByMarketCap)
+
+  const underlyingAssetBalances = useMemo(() => {
+    if (!osmosisOpportunity || !state) return null
+    return getUnderlyingAssetIdsBalances({
+      underlyingAssetIds: osmosisOpportunity.underlyingAssetIds,
+      underlyingAssetRatiosBaseUnit: osmosisOpportunity.underlyingAssetRatiosBaseUnit,
+      cryptoAmountBaseUnit: state.withdraw.shareInAmountBaseUnit,
```

Comment:
No need to tackle it in your PR but since the `shareInAmountBaseUnit` naming was turbo confusing to figure out, let's rename it in a follow-up